xclim/data/fr.json

```
@@ -476,6 +502,13 @@
     "comment": "Formule originale prise de Gladstones 1992.",
     "abstract": "Indice basé sur une relation non linéaire entre la température et les processus physiologiques. Il intègre une borne supérieure du cumul, un coefficient lié à la longueur du jour et une correction liée à l'écart journalier des températures."
   },
+  "LATITUDE_TEMPERATURE_INDEX": {
+    "long_name": "Indice latitude-température calculé avec le facteur de latitude de {lat_factor} (Moyenne de température du mois le plus chaud * (facteur_lat - latitude)).",
+    "description": "Température moyenne du mois le plus chaud avec un facteur d'échelle basé sur la latitude.",
```

```suggestion
    "long_name": "Indice latitude-température",
    "description": "Température moyenne du mois le plus chaud avec un facteur d'échelle basé sur la latitude  : Moyenne de température du mois le plus chaud * ({lat_factor} - latitude).",
```
In order to keep the long name short (:upside_down_face: )

---

I getcha.